source/_integrations/forecast_solar.markdown

```diff
@@ -63,6 +63,8 @@ value.
 
 {% include integrations/config_flow.md %}
 
+If you have more than one plane of solar modules with different properties (e.g. several sides of the roof on different strings or on different buildings, with different directions or declinations) you can add the integration multiple times setting parameters accordingly. You can then use templates to combine the data, e.g. adding up production on different planes. At this point in time, it is not possible to set a combined maximum inverter performance over more then one instance of this integration.
```

Comment:
Why do we need templates?

> At this point in time, it is not possible to set a combined maximum inverter performance over more then one instance of this integration.

This might imply it will be possible in the future? Let's remove this part, as it is not relevant over multiple integration instances.